api/payments.go

```diff
@@ -101,7 +102,7 @@ func (a *API) PaymentCreate(ctx context.Context, w http.ResponseWriter, r *http.
 		return
 	}
 
-	provider := gcontext.GetPaymentProviders(ctx)[params.ProviderType]
+	provider := gcontext.GetPaymentProviders(ctx)[strings.ToLower(params.ProviderType)]
```

Comment:
you'd also want to lower case this when it comes from the config I think, just to be sure that it matches.